### PR TITLE
pair-programming on contact form section with coding partner

### PR DESCRIPTION
**Pull request by admirerbrown**
I collaborated on the contact form section of the project with my coding partner and we did the following changes to the repo:
- I linked the contact form to formspree.io
- i also adjusted the alignment of my submit button
- my coding partner added a HTML 5 form validation to the project
- i implemented 1st code review suggestions.
- added 2nd code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,271 +1,313 @@
...
+            <button class="btn-btn ab-btn" type="button">Terminal</button>
           </li>
         </ul>
       </div>
-      <div class="headline-button">
-        <img src="images/Action.svg" />
+    </section>
+
+    <!-- contact me section -->
+    <section id="abt-contact">
```
Comment (Nemwel-Boniface) [inline index.html:240]: - [ ] Great job styling your form and contact section. However, according to the project requirements, you are supposed to make use of the CSS flexbox to align the items of this section. Kindly do so.

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello  @admirerbrown   :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -238,7 +238,7 @@ <h2 class="ab-title new">Skills</h2>
 
     <!-- contact me section -->
     <section id="abt-contact">
```
Comment (ichala) [inline index.html:240]: - [ ] as mentioned by the previous reviewer, the `contact me` section should use display flex, and not the title , I would suggest adding these CSS styling : 

```CSS
#abt-contact {
    display: flex;
    flex-direction: column;
}
```

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @admirerbrown 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Design ✔️ 


 ## Required Changes ♻️

- [ ] 🛑 **_Check the comments below_**

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
Hi Team! @admirerbrown @billymohajeri, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
Congratulations! 🎉 



## Highlights
✅ Great UI design. 👍🏼 
✅ Great use of GitHub flow. 💯 
✅ Descriptive PR title and summary! 🥇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**